src/core/Global.ttslua

```diff
@@ -645,6 +646,20 @@ function emptyChaosBag()
   end
 end
 
+-- move all regular chaos tokens that are on the table to the chaos bag (e.g. sealed on cards)
+function findAllChaosTokens()
+  local chaosbag = findChaosBag()
+  for _, obj in ipairs(getObjects()) do
+    local name = obj.getName()
+    if tokenChecker.isChaosToken(obj) and
+        not obj.hasTag("tempToken") and
+        name ~= "Bless" and
+        name ~= "Curse" then
+      chaosbag.putObject(obj)
```

Comment:
Cards that seal tokens track those tokens locally.  Just pulling them will leave the state on those incorrect.

Reply:
Limited this function to those sealed tokens